.github/workflows/smoke-test-daily.yml

```diff
@@ -160,59 +213,164 @@ jobs:
+              with:
+                  path: repo
+                  ref: ${{ env.BRANCH_NAME }}
+
+            - name: Download API test report artifact
+              uses: actions/download-artifact@v3
+              with:
+                  name: ${{ env.API_ARTIFACT }}
+                  path: artifacts/api
+
+            - name: Download E2E test report artifact
+              uses: actions/download-artifact@v3
+              with:
+                  name: ${{ env.E2E_ARTIFACT }}
+                  path: artifacts/e2e
+
+            - name: Post test summary
+              uses: actions/github-script@v6
+              env:
+                  API_SUMMARY_PATH: ${{ github.workspace }}/artifacts/api/allure-report/widgets/summary.json
+                  E2E_PW_SUMMARY_PATH: ${{ github.workspace }}/artifacts/e2e/allure-report/widgets/summary.json
+              with:
+                  result-encoding: string
+                  script: |
+                      const script = require( './repo/.github/workflows/scripts/prepare-test-summary-daily.js' )
+                      return await script( { core } )
+
+            - name: Publish report
+              env:
+                  GITHUB_TOKEN: ${{ secrets.REPORTS_TOKEN }}
+                  RUN_ID: ${{ github.run_id }}
+              run: |
+                  gh workflow run publish-test-reports-daily.yml \
+                    -f run_id=$RUN_ID \
+                    -f api_artifact="$API_ARTIFACT" \
+                    -f e2e_artifact="$E2E_ARTIFACT" \
+                    -f s3_root=public \
+                    --repo woocommerce/woocommerce-test-reports
+
+    plugins-results:
+        name: Publish report on smoke tests with plugins
+        if: |
+            always() &&
+            ! github.event.pull_request.head.repo.fork
         runs-on: ubuntu-20.04
-        needs: [login-run, build, test-plugins]
+        needs: [test-plugins]
         env:
             GITHUB_TOKEN: ${{ secrets.REPORTS_TOKEN }}
             RUN_ID: ${{ github.run_id }}
-            API_ARTIFACT: api-test-report---daily
-            COMMIT_MESSAGE: ${{ needs.login-run.outputs.commit_message }}
+            ARTIFACT: Smoke tests with ${{ matrix.plugin }} plugin installed (run ${{ github.run_number }})
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - plugin: 'WooCommerce Payments'
+                      repo: 'automattic/woocommerce-payments'
+                    - plugin: 'WooCommerce PayPal Payments'
+                      repo: 'woocommerce/woocommerce-paypal-payments'
+                    - plugin: 'WooCommerce Shipping & Tax'
+                      repo: 'automattic/woocommerce-services'
+                    - plugin: 'WordPress SEO' # Yoast SEO in the UI, but the slug is wordpress-seo
+                      repo: 'Yoast/wordpress-seo'
+                    - plugin: 'Contact Form 7'
+                      repo: 'takayukister/contact-form-7'
```

Comment:
I did not include WooCommerce Subscriptions for the meantime, since it's a private repo. Its repo URL is a secret, as I can see from [here](https://github.com/woocommerce/woocommerce/blob/e470c62650553a933b615906db82a9139ed14116/.github/workflows/smoke-test-daily.yml#L156), so I'm a bit careful in exposing it even accidentally on the published Allure report. What do you think?

Reply:
Yeah, that's a tricky one.  I can't see anywhere that the URL would show in the Allure reports, but if the test errors out for some reason then it may end up getting dumped there.

What if, for private repo plugins, we suppress the detailed reports and go for more of a pass/fail approach?  Like we can still output the summary stats, but not have the full report generated.  It'll mean a little more work in debugging failures -- that'll all have to happen locally.